src/components/add-movie-button/add-movie-button.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import styles from './add-movie-button.module.scss'
+
+export const AddMovieButton = () => (
+    <button className={styles.addButton}>+ ADD MOVIE</button>
```

Comment:
I am sure you know about that hack: you can easily add `text-transform: uppercase` CSS property to avoid manual uppercase :)